### ISSUE TITLE
Ensure Ubuntu 22.04 compatibility

### ISSUE THREAD

**Issue by stephanosio** [zephyrproject-rtos/sdk-ng#464]
Ubuntu 22.04, a new LTS, has been released.

Verify that the Zephyr SDK is compatible with the Ubuntu 22.04 release and, if not, make any changes necessary to ensure that.

**Comment by stephanosio** [comment]
Following the Zephyr [Getting Started Guide](https://docs.zephyrproject.org/latest/develop/getting_started/index.html), the following issues have been found using Ubuntu 22.04:

1. CMake installation fails using `kitware-archive.sh`:

    > Only Ubuntu Xenial (16.04), Bionic (18.04), and Focal (20.04) are supported. Aborting.

    However, note that the version of CMake provided by the distro is 3.22.1, which is greater than the minimum required version of 3.20.0; so, this step is not absolutely required.

2. Zephyr SDK GDB links against Python 3.8 (default version on Ubuntu 20.04), but the distro default Python version is 3.10.

    * Note that this issue is present on any Ubuntu versions other than 20.04 and is not a critical issue.
    * Python 3.8 is not available on Ubuntu 22.04 from official sources; however, a quick Google search indicates that there are third-party (PPA) packages.

NOTE: Ubuntu 22.04 has the following component versions:
```
# git --version
git version 2.34.1

# ld --version
GNU ld (GNU Binutils for Ubuntu) 2.38

# gcc --version
gcc (Ubuntu 11.2.0-19ubuntu1) 11.2.0

# gperf --version
GNU gperf 3.1

# ccache --version
ccache version 4.5.1

# make --version
GNU Make 4.3

# cmake --version
cmake version 3.22.1

# ninja --version
1.10.1

# python3 --version
Python 3.10.4

# dtc --version
Version: DTC 1.6.1

# dfu-util --version
dfu-util 0.9
```

**Comment by stephanosio** [comment]
No major issues have been found, and no immediate action is necessary to facilitate Zephyr SDK use on Ubuntu 22.04.
